### ISSUE TITLE
Support running in a browser with PGlite

### ISSUE THREAD

**Issue by electrovir** [prisma/prisma#25493]
## Problem

With PGlite working pretty well with the Prisma client now (https://github.com/electrovir/pglite-prisma-demo), I want to run an in-browser Prisma client instance that uses an in-browser memory-only PGlite instance.

At this point in time, this doesn't throw any errors:

```typescript
import {PGlite} from '@electric-sql/pglite';
import {PrismaClient} from '@prisma/client';
import {PrismaPGlite} from 'pglite-prisma-adapter';


const pglite = new PGlite();
await pglite.exec(/* <init-sql-here> */);

const prismaClient = new PrismaClient({
    adapter: new PrismaPGlite(pglite),
});
```

However, when I try to access anything within the client it either hangs forever or prints the "PrismaClient is unable to run in this browser environment" error message.

## Suggested solution

Support running Prisma client in a browser environment when using a PGlite driver.

**Comment by inam-dev** [comment]
Prisma is not meant to run in a Browser. If you want to store something temporarily for a client, consider using localstorage.

Even if it could work in browser, it would be a very heavy tool for a very simple job.

**Comment by jkomyno** [comment]
Hi @electrovir, thanks for opening this issue. We're tracking this one in https://github.com/prisma/prisma/issues/23752, so we will close this duplicate issue.

**Comment by electrovir** [comment]
@jkomyno this issue is not about an official PGlite driver, I’ve already got Prisma working with PGlite. This issue is about browser support.

**Comment by jkomyno** [comment]
Hi @electrovir, I apologise for my oversight. I've re-opened the issue.

**Comment by BjoernRave** [comment]
yeah would be really convenient if someone would have a local pg instance to retrieve data from it in a known way using prisma
